recipes/icu/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+cd source
+chmod +x configure install-sh
+
+if [ "$(uname)" == "Linux" ]; then
+    ./configure --prefix="$PREFIX"
+fi
+
+if [ "$(uname)" == "Darwin" ]; then
+    export LANG=en_US.UTF-8
+    export LC_ALL=en_US.UTF-8
```

Comment:
I think this can be removed (I added these lines to do some tests for a project I was working on last year).